keepassxc-browser/options/options.js

```diff
@@ -276,7 +276,7 @@ options.showKeePassXCVersions = function(response) {
     $('#tab-about span.kpxcVersion').text(response.current);
     $('#tab-general-settings button.checkUpdateKeePassXC:first').attr('disabled', false);
 
-    if (response.current.startsWith('2.6') || response.current === '2.5.3-snapshot') {
+    if (response.current.startsWith('2.6') || response.current.includes('snapshot')) {
```

Comment:
This is also wrong, we want to make sure it _at least starts with 2.6_. We need to convert the version string into a number and then compare it.

Reply:
Ah, true.